src/aae_exchange.erl

```diff
@@ -190,7 +194,7 @@
 -type closing_state() ::
     compare_state()|timeout|error|not_supported.
 -type bucket() :: 
-    {binary(), binary()}|binary().
+    {binary(), binary()}|binary()|all.
```

Comment:
ouch, that's ugly... that works down everywhere

Reply:
Changed name of type to bucket_range()